docs/reference/providers/adobe-commerce.md

```diff
@@ -0,0 +1,14 @@
+# Adobe Commerce
+
+A provider to connect with Adobe Commerce (formerly known as Magento).
+
+## Configuration
+
+To connect to your Adobe Commerce instance, you'll need the following information:
+* Base URL for your Adobe Commerce instance
+* Key present within the URL when editing a product
```

Comment:
Any thoughts on how I can describe this? And an example?

Reply:
Yeah adding an example sounds good. Having never installed an Adobe Commerce Provider I'm not sure what this could be

Reply:
Sounds like it might not be needed. We can take a look and remove it as part of https://atama-team.atlassian.net/browse/PD-1219